python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/resources.py

```diff
@@ -247,20 +260,32 @@ def load_specs(
             return [*asset_specs, *asset_check_specs]
 
     def load_asset_specs(
-        self, dagster_dbt_translator: Optional[DagsterDbtTranslator] = None
+        self,
+        dagster_dbt_translator: Optional[DagsterDbtTranslator] = None,
+        select: str = DBT_CLOUD_DEFAULT_SELECT,
```

Comment:
Specifically I think these arguments should not be optional in internal APIs

Reply:
That's fair - updated in [b3ada91](https://github.com/dagster-io/dagster/pull/28906/commits/b3ada913d860bf24b08bb698497f4a0d486dd126)